zarr/storage.py

```diff
@@ -1038,7 +1038,7 @@ def _normalize_key(self, key):
             key = '/'.join(bits + [end.replace('.', self.key_separator)])
         return key.lower() if self.normalize_keys else key
 
-    def getitems(self, keys):
+    def getitems(self, keys, **kwargs):
```

Comment:
To me it looks like these `**kwargs` are never used. The `on_error="omit"` passed here by `_chunk_getitems` will not be propagated, and instead `on_error="omit"` is hard coded below in the call to `self.map.getitems`.

Reply:
Correct, but the caller now passes `on_error="omit"` to getitems on any mapping. It would make sense to pass kwargs through to the mapping method, but I would argue that anything other than "omit" doesn't make sense here.